packages/components/src/components/text/text.jsx

```diff
@@ -10,28 +10,18 @@ const Text = ({ children, size, color, align, weight, line_height, as, className
             '--text-color': `var(--text-${color || 'general'})`,
             '--text-lh': `var(--text-lh-${line_height || 'm'})`,
             '--text-weight': `var(--text-weight-${weight || 'normal'})`,
+            '--text-align': `var(--text-align-${align || 'start'})`,
         };
         if (!isEmptyObject(styles)) {
             return { ...class_styles, ...styles };
         }
         return class_styles;
     };
 
-    // const style = styles || {
-    //     '--text-size': `var(--text-size-${size || 's'})`,
-    //     '--text-color': `var(--text-${color || 'general'})`,
-    //     '--text-lh': `var(--text-lh-${line_height || 'm'})`,
-    //     '--text-weight': `var(--text-weight-${weight || 'normal'})`,
-    // };
-    const text_align = align || 'start';
-    const class_names = classNames(
-        'dc-text',
-        {
-            [`dc-text--${text_align}`]: !!text_align,
-        },
-        className
-    );
+    const class_names = classNames('dc-text', className);
+    const is_header = as === 'h1' || as === 'h2' || as === 'h3' || as === 'h4' || as === 'h5';
     return (
+        (is_header && React.createElement(as, { className: class_names, style: setStyle(), ...props }, children)) ||
```

Comment:
Is it possible to remove all these conditionals and only use: 
`React.createElement(as, { className: class_names, style: setStyle(), ...props }, children))`?